src/components/ResultsView.tsx

```diff
@@ -286,10 +293,17 @@ const ResultsView: ResultsViewComponent = ({
       );
   };
   const debounceRef = useRef(0);
+  const minimal = layout.mode === "minimal" ? true : false;
+  const [showInterface, setShowInterface] = useState(minimal ? false : true);
+  const [showIcons, setShowIcons] = useState(minimal ? false : true);
+  const appear = useCallback(() => setShowIcons(true), [setShowIcons]);
+  const disappear = useCallback(() => setShowIcons(false), [setShowIcons]);
```

Comment:
In general, we want to start moving state that is specific to only a single layout into its layout comonent. For now, this is okay and can be revisited in the future

Reply:
Would that then entail moving the controls to each layout?